### PR TITLE
Fix Align Left Account Menu

### PR DESCRIPTION
**Pull request by carpanese**
In the Bootstrap update, the login screen, account and password for example are getting off the screen. A scroll bar is generated below.

This improvement leaves everything correct. I ran the tests on the new generator and it worked perfectly.

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/layouts/navbar/_navbar.scss [hunk 1]
```diff
@@ -57,6 +57,9 @@ Navbar
             color: #fff;
         }
     }
+	.dropdown-menu-right {
```
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/layouts/navbar/_navbar.scss]: I don't think this is necessary. Adding `.ml-auto` to the element will fix it.

**Comment by deepu105** [comment]
@carpanese can you make the requested change?

**Comment by deepu105** [comment]
@sendilkumarn if there is no response could you update this?

**Comment by sendilkumarn** [comment]
closing this due to inactivity

**Pull request closed without merging**